Nest/00. Plan.md

```diff
@@ -0,0 +1,53 @@
+# 1 Introduction
+## 1.1 What is a package manager?
+
+## 1.2 Packages and dependencies
+
+## 1.3 Repositories
+
+## 1.4 Package's syntax
+
+# 2 Nest
+## 2.1 Nest's options
+
+## 2.2 Nest commands
+### 2.2.1 Updating the list of available packages
+
+### 2.2.2 Installing and uninstalling a package
+
+### 2.2.3 Updating the system
+
+## 2.3 Summary
+
+# 3 Advanced concepts
+## 3.1 Managing repositories
+### 3.1.1 The official repositories
+### 3.1.2 The unofficial repositories
+### 3.1.3 Configuring repositories
+
+## 3.2 Overview of the cache
+
+## 3.3 Version requirement
+
+# 4 Finest
+## 4.1 Key concepts
+### 4.1.1 The cache in detail
+### 4.1.2 The transactions
```

Comment:
```suggestion
### 4.1.2 Transactions
```
I don't think there's a specific set of transactions right?

Reply:
(Actually there is (`PULL`, `INSTALL`,`UNINSTALL` and `UPDATE`), but `Transactions` is still a better name :D)

Reply:
`Transactions` sounds better indeed. `The transactions` seem like I'm going to list them all in this section (the point of this section is rather to explain the concept of transactions than listing them)